.github/increment_version.py

```diff
@@ -155,14 +156,14 @@ def update_metadata_files(version: str) -> bool:
     if arduino_meta_file.exists():
         # simple search and replace
         ver_pattern = re.compile(r"version=(\d+\.\d+\.?\d*)")
-        data = arduino_meta_file.read_text(encoding="utf-8")
-        ver_match = ver_pattern.search(data)
+        ini_data: str = arduino_meta_file.read_text(encoding="utf-8")
```

Comment:
don't need the type hint here
```suggestion
        ini_data = arduino_meta_file.read_text(encoding="utf-8")
```